osu.Game/Overlays/OverlayColourProvider.cs

```diff
@@ -57,56 +55,22 @@ public void ChangeColourScheme(OverlayColourScheme colourScheme)
             ColourScheme = colourScheme;
         }
 
-        private Color4 getColour(float saturation, float lightness) => Color4.FromHsl(new Vector4(getBaseHue(ColourScheme), saturation, lightness, 1));
+        private Color4 getColour(float saturation, float lightness) => Framework.Graphics.Colour4.FromHSL(getBaseHue(ColourScheme), saturation, lightness);
 
-        // See https://github.com/ppy/osu-web/blob/5a536d217a21582aad999db50a981003d3ad5659/app/helpers.php#L1620-L1628
-        private static float getBaseHue(OverlayColourScheme colourScheme)
-        {
-            switch (colourScheme)
-            {
-                default:
-                    throw new ArgumentException($@"{colourScheme} colour scheme does not provide a hue value in {nameof(getBaseHue)}.");
-
-                case OverlayColourScheme.Red:
-                    return 0;
-
-                case OverlayColourScheme.Pink:
-                    return 333 / 360f;
-
-                case OverlayColourScheme.Orange:
-                    return 45 / 360f;
-
-                case OverlayColourScheme.Lime:
-                    return 90 / 360f;
-
-                case OverlayColourScheme.Green:
-                    return 125 / 360f;
-
-                case OverlayColourScheme.Aquamarine:
-                    return 160 / 360f;
-
-                case OverlayColourScheme.Purple:
-                    return 255 / 360f;
-
-                case OverlayColourScheme.Blue:
-                    return 200 / 360f;
-
-                case OverlayColourScheme.Plum:
-                    return 320 / 360f;
-            }
-        }
+        private static float getBaseHue(OverlayColourScheme colourScheme) => (int)colourScheme / 360f;
     }
 
+    // See https://github.com/ppy/osu-web/blob/5a536d217a21582aad999db50a981003d3ad5659/app/helpers.php#L1620-L1628
     public enum OverlayColourScheme
     {
-        Red,
-        Pink,
-        Orange,
-        Lime,
-        Green,
-        Purple,
-        Blue,
-        Plum,
-        Aquamarine
+        Red = 0,
+        Orange = 45,
+        Lime = 90,
+        Green = 125,
+        Aquamarine = 160,
+        Blue = 200,
+        Purple = 255,
+        Plum = 320,
+        Pink = 333,
```

Comment:
While I'm fine with it I did see @peppy [express disdain at giving enums numerical values like that](https://discord.com/channels/188630481301012481/188630652340404224/1235237547865673863).

Reply:
Sounds good.